packages/lb-components/src/components/pointCloudView/hooks/useStatus.ts

```diff
@@ -55,47 +55,45 @@ export const useStatus = () => {
       return;
     }
 
+    const instanceArr = [topViewInstance, sideViewInstance, backViewInstance]
+
     switch (toolName) {
       case EToolName.Rect:
         sideViewInstance?.toolInstance.clearResult()
         backViewInstance?.toolInstance.clearResult()
 
-        topViewInstance?.switchToCanvas(EToolName.PointCloudPolygon)
-        topViewInstance?.toolInstance.setPattern(EPolygonPattern.Rect)
-        sideViewInstance?.switchToCanvas(EToolName.PointCloudPolygon)
-        sideViewInstance?.toolInstance.setPattern(EPolygonPattern.Rect)
-        backViewInstance?.switchToCanvas(EToolName.PointCloudPolygon)
-        backViewInstance?.toolInstance.setPattern(EPolygonPattern.Rect)
+        instanceArr.forEach((instance) => {
+          instance?.switchToCanvas(EToolName.PointCloudPolygon)
+          instance?.toolInstance.setPattern(EPolygonPattern.Rect)
+        })
         setPointCloudPattern(EToolName.Rect);
         break;
       case EToolName.Polygon:
         sideViewInstance?.toolInstance.clearResult()
         backViewInstance?.toolInstance.clearResult()
```

Comment:
You can extract it. 
``` 
sideViewInstance?.toolInstance.clearResult()
backViewInstance?.toolInstance.clearResult()
```
All cases need to call this function

Reply:
done